nsv13/code/modules/overmap/types/syndicate.dm

```diff
@@ -375,6 +375,45 @@
 	weapon_types[FIRE_MODE_GAUSS] = new /datum/ship_weapon/gauss(src) //AI ships want to be able to use gauss too. I say let them...
 	weapon_types[FIRE_MODE_MISSILE] = new/datum/ship_weapon/missile_launcher(src)
 
+/obj/structure/overmap/syndicate/ai/kadesh	//I sure wonder what this one does....
+	name = "Kadesh class advanced cruiser"
+	desc = "An experimental Syndicate cruiser capable of projecting energy bursts powerful enough to disrupt drive systems."
+	obj_integrity = 2250	//Pretty thick hull due to it being a priority target
+	max_integrity = 2250
+	integrity_failure = 2250
+	icon = 'nsv13/icons/overmap/syndicate/syn_light_cruiser.dmi'
+	icon_state = "advanced_cruiser"
+	damage_states = FALSE	//Maybe later
+	sprite_size = 96
+	bound_height = 128
+	bound_width = 128
+	armor = list("overmap_light" = 90, "overmap_heavy" = 25)
```

Comment:
You're now going to want to add overmap_medium to this aswell, and I think 2250 integrity is way too much, maybe make it 1500-1750 max?

Reply:
Aight.